prow/plugins/yuks/yuks.go

```diff
@@ -115,18 +129,18 @@ func handle(gc githubClient, log *logrus.Entry, e *github.GenericCommentEvent, j
 	repo := e.Repo.Name
 	number := e.Number
 
-	for i := 0; i < 10; i++ {
+	errorBudget := 5
+	for i := 1; i <= errorBudget; i++ {
 		resp, err := j.readJoke()
-		if err != nil {
-			return err
-		}
-		if simple.MatchString(resp) {
-			log.Infof("Commenting with \"%s\".", resp)
-			return gc.CreateComment(org, repo, number, plugins.FormatResponseRaw(e.Body, e.HTMLURL, e.User.Login, resp))
+		if err != nil || resp == "" {
+			log.WithError(err).Printf("failed to get joke. Retrying (%d/%d)", i, errorBudget)
```

Comment:
Could use a `Info` or `Debug` level here instead of `Printf`?

Reply:
Changed it to `Infof`